packages/local-transport.js

```diff
@@ -0,0 +1,11 @@
+/*
+ * Copyright 2017 The boardgame.io Authors
+ *
+ * Use of this source code is governed by a MIT-style
+ * license that can be found in the LICENSE file or at
+ * https://opensource.org/licenses/MIT.
+ */
+
+import { Local } from '../src/client/transport/local.js';
```

Comment:
Let's put all the transport implementations under one package `boardgame.io/multiplayer`.  Modern bundlers will tree shake the stuff you don't import.